src/schema/objects/suffixes.yaml

```
@@ -670,6 +670,23 @@ nirs:
   value: nirs
   display_name: Near Infrared Spectroscopy
   description: Data associated with a Shared Near Infrared Spectroscopy Format file.
+noRF:
+  value: noRF
+  display_name: No Radio Frequency Excitation Scan
+  description: |
+    An MR noise scan often acquired alongside another MR scan.
+
+    These noise scans are acquired by disabling the radio frequency excitation,
+    while maintaining all other parameters from the associated scan.
+    These can be aquired using some other sequence with RF excitation removed
+    or as part of sequences that turns off RF at the beginning or end of a run.
+    For example a `bold` scan where a few volumes are acquired with RF off.
+    There is no consistent DICOM denotation for these scans so separating and
+    labeling noRF volumes would need to be done outside of automatic DICOM
+    conversion.
+
+    `noRF` are typically used to estimate properties of the scanner and
```

Echoing discussion, if decide to go with the suffix, might be valuable to say how to distinguish in case of conflicts, e.g. smth like

```suggestion
    If multiple sequences with different suffixes but otherwise the same entities have `noRF`
    sequences, disambiguate using modality entity, e.g. `mod-T1w_noRF.nii` and `mod-T2w_noRF.nii`.
    
    `noRF` are typically used to estimate properties of the scanner and
```